src/bun.js/bindings/JSCell.zig

```diff
@@ -1,47 +1,41 @@
 const std = @import("std");
 const bun = @import("root").bun;
+const GetterSetter = @import("GetterSetter.zig").GetterSetter;
+const CustomGetterSetter = @import("CustomGetterSetter.zig").CustomGetterSetter;
+const FFI = @import("FFI.zig");
 const JSC = bun.JSC;
 const JSValue = JSC.JSValue;
 const JSGlobalObject = JSC.JSGlobalObject;
-const GetterSetter = @import("GetterSetter.zig").GetterSetter;
-const CustomGetterSetter = @import("CustomGetterSetter.zig").CustomGetterSetter;
 
 pub const JSCell = extern struct {
-    pub const shim = JSC.Shimmer("JSC", "JSCell", @This());
-    bytes: shim.Bytes,
-    const cppFn = shim.cppFn;
-    pub const include = "JavaScriptCore/JSCell.h";
-    pub const name = "JSC::JSCell";
-    pub const namespace = "JSC";
-
-    const CellType = enum(u8) { _ };
-
     /// Statically cast a cell to a JSObject. Returns null for non-objects.
     /// Use `toObject` to mutate non-objects into objects.
     pub fn getObject(this: *JSCell) ?*JSC.JSObject {
-        return shim.cppFn("getObject", .{this});
+        JSC.markMemberBinding(JSCell, @src());
+        return JSC__JSCell__getObject(this);
     }
 
     /// Convert a cell to a JSObject.
     ///
     /// Statically casts cells that are already objects, otherwise mutates them
     /// into objects.
+    ///
+    /// ## References
+    /// - [ECMA-262 §7.1.18 ToObject](https://tc39.es/ecma262/#sec-toobject)
     pub fn toObject(this: *JSCell, global: *JSC.JSGlobalObject) *JSC.JSObject {
```

Comment:
This can throw, in which case it will return a null pointer

Reply:
`cell->toObject` cannot throw. `null` and `undefined` have already been ruled out

Reply:
^